src/jsonUtils/computeNode.js

```diff
@@ -0,0 +1,457 @@
+      } else {
+        yogaNode.setMargin(yoga.EDGE_HORIZONTAL, style.marginHorizontal);
+      }
+    }
+    if (style.margin) {
+      if (isPercent(style.margin)) {
+        yogaNode.setMarginPercent(yoga.EDGE_ALL, style.margin);
+      } else if (isAuto(style.margin)) {
+        yogaNode.setMarginAuto(yoga.EDGE_ALL);
+      } else {
+        yogaNode.setMargin(yoga.EDGE_ALL, style.margin);
+      }
+    }
+
+    // Padding
+    if (style.paddingTop) {
+      if (isPercent(style.paddingTop)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_TOP, style.paddingTop);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_TOP, style.paddingTop);
+      }
+    }
+    if (style.paddingBottom) {
+      if (isPercent(style.paddingBottom)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_BOTTOM, style.paddingBottom);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_BOTTOM, style.paddingBottom);
+      }
+    }
+    if (style.paddingLeft) {
+      if (isPercent(style.paddingLeft)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_LEFT, style.paddingLeft);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_LEFT, style.paddingLeft);
+      }
+    }
+    if (style.paddingRight) {
+      if (isPercent(style.paddingRight)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_RIGHT, style.paddingRight);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_RIGHT, style.paddingRight);
+      }
+    }
+    if (style.paddingVertical) {
+      if (isPercent(style.paddingVertical)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_VERTICAL, style.paddingVertical);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_VERTICAL, style.paddingVertical);
+      }
+    }
+    if (style.paddingHorizontal) {
+      if (isPercent(style.paddingHorizontal)) {
+        yogaNode.setPaddingPercent(
+          yoga.EDGE_HORIZONTAL,
+          style.paddingHorizontal
+        );
+      } else {
+        yogaNode.setPadding(yoga.EDGE_HORIZONTAL, style.paddingHorizontal);
+      }
+    }
+    if (style.padding) {
+      if (isPercent(style.padding)) {
+        yogaNode.setPaddingPercent(yoga.EDGE_ALL, style.padding);
+      } else {
+        yogaNode.setPadding(yoga.EDGE_ALL, style.padding);
+      }
+    }
+
+    // Border
+    if (style.borderTop) {
+      yogaNode.setBorder(yoga.EDGE_TOP, style.borderTop);
+    }
+    if (style.borderBottom) {
+      yogaNode.setBorder(yoga.EDGE_BOTTOM, style.borderBottom);
+    }
+    if (style.borderLeft) {
+      yogaNode.setBorder(yoga.EDGE_LEFT, style.borderLeft);
+    }
+    if (style.borderRight) {
+      yogaNode.setBorder(yoga.EDGE_RIGHT, style.borderRight);
+    }
+    if (style.borderVertical) {
+      yogaNode.setBorder(yoga.EDGE_VERTICAL, style.borderVertical);
+    }
+    if (style.borderHorizontal) {
+      yogaNode.setBorder(yoga.EDGE_HORIZONTAL, style.borderHorizontal);
+    }
+    if (style.border) {
+      yogaNode.setBorder(yoga.EDGE_ALL, style.border);
+    }
+
+    // Flex
+    if (style.flex) {
```

Comment:
We need to explicitly check for `undefined` in these, so we don't ignore zero values

Reply:
Good call. Fixed